content/_code-samples/secure-signing/java/SignPayment.java

```diff
@@ -0,0 +1,72 @@
+////////////////////////////////////////////////////////////////////////////
```

Comment:
The signing interface in xrpl4j is very different than that of the python and js libraries to more easily extend to supporting HSM signing. We currently have two implementations of `SignatureService`, aka two ways to sign transactions out of the box in xrpl4j.  I provided example code for both in this code snippet so as not to introduce another section of the tutorial, but happy to hear pushback on this.